hugo/layouts/company/single.html

```diff
@@ -495,6 +495,16 @@ <h4 class="text-dark title-line line-yellow mt-3">Fernanda Gomes</h4>
                     </div>
                 </div>
             </div>
+            <div class="col-sm-6 col-lg-3 mb-4">
+                <div class="card white">
+                    <img class="w-100" src="assets/img/team/alicia.png" alt="">
+                    <div class="card-body">
+                        <a href="https://www.linkedin.com/in/alicia-kozlowski/" class="ic icon-lin text-light"></a>
+                        <h4 class="text-dark title-line line-yellow mt-3">Alicia Kozlowski</h4>
+                        <span class="text-medium">Intern Product Designer</span>
```

Comment:
I believe "Intern UX Researcher" would be more appropriate. @marnovo do you agree?

Reply:
@aliciakozlowski @ricardobaeta 
`Intern Product Designer` is fine, follows suit with the patterns we have on product and engineering teams. Let's not overcomplicate, and her work stems beyond user research… this PR and the work on the 404 page are examples of that. ;)